pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -90,134 +90,148 @@ func TestAssumePodScheduled(t *testing.T) {
 	}
 
 	tests := []struct {
-		pods []*v1.Pod
-
+		pods      []*v1.Pod
+		name      string
 		wNodeInfo *NodeInfo
-	}{{
-		pods: []*v1.Pod{testPods[0]},
-		wNodeInfo: &NodeInfo{
-			requestedResource: &Resource{
-				MilliCPU: 100,
-				Memory:   500,
-			},
-			nonzeroRequest: &Resource{
-				MilliCPU: 100,
-				Memory:   500,
+	}{
+		{
+			name: fmt.Sprintf("%v", testPods[0].ObjectMeta.Name),
```

Comment:
I think these tests could have more meaningful names. If you're not sure what is actually being exercised in each test, let's chat about it. I don't think it's that obvious.

Reply:
agreed,
if you could give some suggestions/guidance on meaningful and clear names that would help a lot :)